hera_cal/noise.py

```diff
@@ -17,12 +17,13 @@
 
 
 def interleaved_noise_variance_estimate(vis, kernel=[[1, -2, 1], [-2, 4, -2], [1, -2, 1]]):
```

Comment:
Wait, is this differencing auto correlations, visibilities, or both?

Reply:
It's meant for visibilities, but in theory you could find the "noise" on autocorrelations using it.